DEPLOYMENT_TYPES.md

```diff
@@ -1,10 +1,11 @@
-#Deployment Types
+# Deployment Types
 
-This module supports OpenShift Origin, OpenShift Enterprise, and Atomic
-Enterprise Platform. Each deployment type sets various defaults used throughout
-your environment.
+This repository supports OpenShift Origin and OpenShift Container Platform. Previous product versions were also called OpenShift Enterprise and Atomic
+Enterprise Platform.
```

Comment:
Is it fair to mention the old names? I'd be fine with cleaning it up and having only current names.